elide-core/src/main/java/com/yahoo/elide/utils/ClassScanner.java

```diff
@@ -57,6 +58,18 @@
         }
     }
 
+    /**
+     * Scans all classes accessible from the context class loader which belong to the current class loader.
+     *
+     * @param annotation  Annotation to search
+     * @return The classeInfos
+     */
+    static public List<ClassInfo> getAnnotatedClassInfos(Class<? extends Annotation> annotation) {
```

Comment:
This method leaks ClassInfo.  We don't want ClassGraph concepts to leak into core Elide.  I would just use getAnnotatedClasses instead.

Reply:
It slipped through. Its not used anywhere. Will simply remove it.